src/transformers/models/detr/modeling_detr.py

```diff
@@ -320,45 +322,56 @@ def replace_batch_norm(m, name=""):
         replace_batch_norm(ch, n)
 
 
-class DetrTimmConvEncoder(nn.Module):
+class DetrBackbone(nn.Module):
```

Comment:
I'm wondering about the class name here as it's not a backbone in the same sense as the `ResnetBackbone`. I'd maybe stick to something like `DetrConvEncoder`

Reply:
Yeah the reason I dropped "Conv" is because more models are coming out which don't necessarily use a convolutional model as backbone (but rather a Swin Transformer for instance)

Reply:
Ok as DETR only has pre-trained weights with convolutional backbones, I've named it `DetrConvEncoder`.